examples/qtmegachatapi/chatWindow.cpp

```diff
@@ -145,9 +145,22 @@ void ChatWindow::onMessageUpdate(megachat::MegaChatApi* api, megachat::MegaChatM
     {
         if (chatMessage)
         {
-            chatMessage->setMessageContent(msg->getContent());
-            if (msg->isEdited())
-                chatMessage->markAsEdited();
+            if (msg->getType() == megachat::MegaChatMessage::TYPE_TRUNCATE)
+            {
+                truncateChatUI();
+                megachat::MegaChatMessage *auxMsg = msg->copy();
+                addMsgWidget(auxMsg, loadedMessages);
+                mChatItemWidget->setOlderMessageLoaded(msg->getMsgId());
+                mMegaChatApi->setMessageSeen(mChatRoom->getChatId(), msg->getMsgId());
```

Comment:
In case the truncate is ours, the app should NOT set the message as seen. It doesn't have any consequence, since MEGAchat already checks the `userid`, but better to check at the app.